### PR TITLE
[CFM-1266] New Project button

### PR DESCRIPTION
**Pull request by guytepper**
JIRA: https://issuetracker.amplexor.com/jira/browse/CFM-1266

### CODE REVIEW

**Comment by guytepper** [comment]
## Homepage
### Mobile
There seems to be some issue with the mobile menu - 
homepage_mobile-issue

### Tablet
image

### Desktop
image


## Project Overview
### Mobile
image

### Tablet
image

### Desktop
image

File: project/profiles/capacity4more/modules/c4m/content/c4m_content_project/c4m_content_project.block.inc [hunk 1]
```diff
@@ -13,7 +13,7 @@ function _c4m_content_project_block_c4m_block_create_project() {
 
   if ($user->uid) {
     if (user_access('create project content')) {
-      $markup = l(t('Create a project'), 'node/add/project', array('attributes' => array('class' => array('btn', 'btn-default'))));
+      $markup = '<div class="c4m-group-visitor-cta">' . l(t('Create a project'), 'node/add/project', array('attributes' => array('class' => array('btn', 'btn-default')))) . '</div>';
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/content/c4m_content_project/c4m_content_project.block.inc]: @guytepper 
It's better to add a specific class to the link and use the same styling

**Comment by guytepper** [comment]
New button was added in #1129. I've applied the new button design to it:

## Mobile
image

## Tablet
image

## Desktop 
image

**Comment by ordavidil** [comment]
@guytepper 
please re-roll

File: project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/components/_groups.scss [hunk 2]
```diff
@@ -819,22 +819,6 @@
...
-  a.btn {
-    color: $color-white;
-    background-color: $color-text-alternative;
-    margin: 10px 0;
-    white-space: normal;
-    width: 100%;
-  }
-}
-
 .c4m-group-invite-cta {
```
Comment (ordavidil) [inline project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/components/_groups.scss]: @guytepper 
shouldn't we remove this one as well ?

Comment (guytepper) [inline project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/components/_groups.scss]: Nope -  
screen shot 2016-08-30 at 3 31 48 pm
screen shot 2016-08-30 at 3 31 54 pm

**Pull request merged**